bio/star/align/wrapper.py

```
@@ -45,10 +45,14 @@
 if out_unmapped:
     out_unmapped = "--outReadsUnmapped Fastx"
 
-index = snakemake.input.get("idx")
-if not index:
-    index = snakemake.params.get("idx", "")
-
+genome_dir = path.commonpath(snakemake.input.get("idx"))
```

Ruff (0.15.9)

[error] Undefined name `snakemake`

(F821)